workers/unity/Assets/Gdk/Core/Worker/WorkerBase.cs

```diff
@@ -71,7 +75,7 @@ public virtual void RegisterSystems()
             RegisterCoreSystems();
         }
 
-        protected void RegisterCoreSystems()
+        private void RegisterCoreSystems()
```

Comment:
Why the change?

Reply:
As this is only called in a virtual function it should be protected. Arguably it should be called from a non virtual function that then calls a virtual one.

Reply:
Ah I see, missed that. Reverted.